src/Target.cpp

```diff
@@ -1385,9 +1393,17 @@ bool Target::get_runtime_compatible_target(const Target &other, Target &result)
     // clang-format on
 
     // clang-format off
-    const std::array<Feature, 15> intersection_features = {{
+    const std::array<Feature, 23> intersection_features = {{
         ARMv7s,
         ARMv81a,
+        ARMv82a,
+        ARMv83a,
+        ARMv84a,
+        ARMv85a,
+        ARMv86a,
+        ARMv87a,
+        ARMv88a,
+        ARMv89a,
```

Comment:
Aren't these more like the CUDA Capability / HVX features where we need to gather all the enabled flags and then take the meet?

Reply:
> take the meet

?

Reply:
The lower bound. Like if you have both armv8.6 and armv8.4 then the runtime should use armv8.4

Reply:
Ah, yes, stand by